### PR TITLE
Wysiwym link modal fix

### PR DESCRIPTION
**Pull request by jpuri**
PR addresses issue of url input in link modal not focused when adding link.

It also fixes a small issue in adding row at position 0 in table.

closes: https://github.com/tinacms/tinacms/issues/689

### CODE REVIEW

File: packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/FloatingTableMenu/AddRowMenu.tsx [hunk 1]
```diff
@@ -38,6 +43,19 @@ export default ({ index, marker, tableWidth, view }: AddRowMenuProps) => {
...
+      if (!tableNode) return
+      const tableMap = TableMap.get(tableNode.node)
+      const position = tableNode.start + tableMap.map[tableMap.width] - 1
+      const cellInNextRow = getCellsInRow(0)(state.selection)
+      if (!cellInNextRow) return
+      const cells = cellInNextRow?.map(cell =>
+        table_cell.createAndFill({ ...cell.node.attrs })
+      )
+      dispatch(state.tr.insert(position, table_row.create(null, cells)))
+    }
```
Comment (jpuri) [inline packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/FloatingTableMenu/AddRowMenu.tsx]: Inserting row at position `0` in table had issue this fixes it.

File: packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/FloatingTableMenu/FloatingTableColumnMenu.tsx [hunk 2]
```diff
@@ -41,7 +41,8 @@ const alignColumn = (view: EditorView, alignValue: string) => {
   const tableHeader = findParentNodeOfType(table_header)(state.selection)
```
Comment (jpuri) [inline packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/FloatingTableMenu/FloatingTableColumnMenu.tsx]: This file has only prettier changes.

File: packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/markControl.tsx [hunk 3]
```diff
@@ -98,8 +98,6 @@ export function markControl({
       }
 
       toggleMark(this.markType(mark), defaultAttrs)(state, dispatch)
```
Comment (jpuri) [inline packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/markControl.tsx]: Following change is to fix focus in link modal.

File: packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/FloatingTableMenu/AddRowMenu.tsx (hunk 1 above)
Comment (DirtyF) [reply packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/FloatingTableMenu/AddRowMenu.tsx]: Successfully tested locally

File: packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/markControl.tsx (hunk 3 above)
Comment (DirtyF) [reply packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/markControl.tsx]: Successfully tested locally

**Pull request merged**